src/table/table-role/__integ__/grid-navigation.test.ts

```diff
@@ -109,11 +109,11 @@ test(
   setupTest({ actionsMode: 'inline' }, async page => {
     await page.click('[data-testid="link-before"]');
     await page.keys('Tab');
-    await page.keys(['ArrowRight', 'ArrowDown', 'ArrowRight', 'ArrowRight']);
-    await expect(page.isFocused('tr[aria-rowindex="2"] button[aria-label="Update item"]')).resolves.toBe(true);
+    await page.keys(['ArrowRight', 'ArrowDown', 'ArrowDown', 'ArrowRight', 'ArrowRight']);
+    await expect(page.isFocused('tr[aria-rowindex="3"] button[aria-label="Update item"]')).resolves.toBe(true);
```

Comment:
Same as above: small changes were required to account for the newly generated data.